ws-server/src/tests.rs

```diff
@@ -653,7 +653,7 @@ async fn custom_subscription_id_works() {
 	module
 		.register_subscription("subscribe_hello", "subscribe_hello", "unsubscribe_hello", |_, sink, _| {
 			std::thread::spawn(move || loop {
-				let _ = sink;
+				let _ = &sink;
```

Comment:
required by https://doc.rust-lang.org/edition-guide/rust-2021/disjoint-capture-in-closures.html#wild-card-patterns